webapp/javascript/components/TimelineChart/AnnotationMark/index.tsx

```diff
@@ -0,0 +1,65 @@
+import Color from 'color';
+import React, { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import AnnotationInfo from '@webapp/pages/continuous/contextMenu/AnnotationInfo';
+import useTimeZone from '@webapp/hooks/timeZone.hook';
+import { faCommentDots } from '@fortawesome/free-regular-svg-icons/faCommentDots';
+
+import styles from './styles.module.scss';
+
+interface IAnnotationMarkProps {
+  type: 'message';
+  color: Color;
+  value: {
+    content: string;
+    timestamp: number;
+  };
+}
+
+const getIcon = (type: IAnnotationMarkProps['type']) => {
+  switch (type) {
+    case 'message':
+    default:
+      return faCommentDots;
+  }
+};
+
+const AnnotationMark = ({ type, color, value }: IAnnotationMarkProps) => {
+  const { offset } = useTimeZone();
+  const [visible, toggle] = useState(false);
```

Comment:
```suggestion
  const [visible, setVisible] = useState(false);
```

just to make things more explicit